autonomy/deploy/base.py

```diff
@@ -315,12 +379,103 @@ def try_update_runtime_params(
                     skill_id=component_id.public_id,
                     has_multiple_overrides=has_multiple_overrides,
                 )
+                self._try_update_tendermint_params(
+                    override=override,
+                    skill_id=component_id.public_id,
+                    has_multiple_overrides=has_multiple_overrides,
+                )
 
             override["type"] = component_id.package_type.value
             override["public_id"] = str(component_id.public_id)
 
         self.service.overrides = overrides
 
+    def _update_abci_connection_config(
+        self,
+        overrides: Dict,
+        has_multiple_overrides: bool = False,
+    ) -> Dict:
+        """Update ABCI connection config"""
```

Comment:
Method to update ABCI connection config

The docker compose build

```yaml
      - CONNECTION_ABCI_CONFIG_HOST=abci0
      - CONNECTION_ABCI_CONFIG_PORT=26658
```

and the kubernetes build

```yaml
        - name: CONNECTION_ABCI_CONFIG_HOST
          value: localhost
        - name: CONNECTION_ABCI_CONFIG_PORT
          value: '26658'
```